.github/workflows/python.yaml

```diff
@@ -0,0 +1,19 @@
+name: flake8 Lint
+
+on: [push, pull_request]
```

Comment:
Note that this runs on both `push` and `pull_request`. Not sure if that's warranted, or we want `pull_request` only.

Reply:
Probably this is only necessary for PRs and commits in the `master` branch.